plugin/accent.vim

```diff
@@ -1,24 +1,28 @@
 scriptencoding utf-8
-
 if exists("g:loaded_accent") || &cp
   finish
 endif
 let g:loaded_accent = 1
 
 let s:lists = [
-  \ ['a','ä','à','â'],
-  \ ['A','Ä','À','Â'],
-  \ ['e','é','è','ê','ë'],
-  \ ['E','É','Ê','È'],
-  \ ['i','î','ï'],
-  \ ['I','Î'],
-  \ ['o','ö','ô'],
-  \ ['O','Ö','Ô'],
+  \ ['a','ä','à','â','á'],
+  \ ['A','Ä','À','Â','Á'],
+  \ ['e','é','è','ê','ë','é'],
+  \ ['E','É','Ê','È','É'],
```

Comment:
Is this accent different from the second item in the list?

Reply:
oh, yeah, I guess it is. I'll update.